dev/deps/spark-deps-hadoop-2.7

```diff
@@ -20,6 +20,7 @@ arpack_combined_all-0.1.jar
 arrow-format-0.12.0.jar
 arrow-memory-0.12.0.jar
 arrow-vector-0.12.0.jar
+audience-annotations-0.5.0.jar
```

Comment:
This is an ALv2 dependency from Yetus, it seems. It needs a line in LICENSE-binary with other ASF dependencies. We'd also have to check if it has a NOTICE file bundled to append to NOTICE-binary

Reply:
@srowen Thanks very much! However, I have not fully understood what you mean. Do we need NOTICE file?

Reply:
I checked the `audience-annotations-0.5.0.jar` and find the NOTICE file.
The content of the NOTICE file show below:

```
Apache Yetus - Audience Annotations
Copyright 2015-2017 The Apache Software Foundation

This product includes software developed at
The Apache Software Foundation (http://www.apache.org/).
```

Reply:
That's fine, just append this to the bottom of `NOTICE-binary`

Reply:
OK. I will append the `NOTICE-binary`.